src/main/java/org/folio/service/titles/TitlesService.java

```diff
@@ -45,13 +55,21 @@ public TitlesService(RestClient restClient, AcquisitionsUnitsService acquisition
   }
 
   public Future<Title> createTitle(Title title, RequestContext requestContext) {
-    return inventoryManager.createShadowInstanceIfNeeded(title.getInstanceId(), requestContext)
+    return validateTitleAcqUnitOnCreate(title.getAcqUnitIds(), requestContext)
+      .compose(v -> inventoryManager.createShadowInstanceIfNeeded(title.getInstanceId(), requestContext))
       .compose(shadowInstance -> {
         RequestEntry requestEntry = new RequestEntry(ENDPOINT);
         return restClient.post(requestEntry, title, Title.class, requestContext);
       });
   }
 
+  private Future<Void> validateTitleAcqUnitOnCreate(List<String> acqUnitIds, RequestContext requestContext) {
```

Comment:
We have similar method in PurchaseOrderHelper validateAcqUnitsOnCreate that should be reused insted of creating new one, also that method also checks that acq unit active, and this new method skips this check, we should have the similar behavior

Reply:
made code refactoring to invoke the same method for both places, for orders and titles